src/transaction/log_manager.c

```diff
@@ -1409,6 +1409,24 @@ log_initialize_internal (THREAD_ENTRY * thread_p, const char *db_fullname, const
     }
   log_Gl.rcv_phase = LOG_RESTARTED;
 
+  // PREREQ: metalog is loaded at this point
+  if (is_tran_server_with_remote_storage ())
```

Comment:
```suggestion
  if (!log_Gl.m_metainfo.is_tsrs_shutdown () && is_tran_server_with_remote_storage ())
```

Reply:
Isn't it even better as:
```suggestion
  if (log_Gl.m_metainfo.is_tsrs_shutdown () && is_tran_server_with_remote_storage ())
```
..in which case the writing of the metalog would be avoided if flag is already `false`?